source/user-manual/reference/tools/wazuh-control.rst

```diff
@@ -1,11 +1,13 @@
 .. Copyright (C) 2020 Wazuh, Inc.
 
-.. _ossec-control:
+.. _wazuh-control:
 
-ossec-control
+wazuh-control
 =============
 
-The ossec-control script is used to start, stop, configure, or check on the status of Wazuh processes. This script can enable or disable client-syslog, database logging, agentless configurations, integration with slack and pagerduty, and debug mode.
+.. versionadded:: 4.2
+
+The wazuh-control script is used to start, stop, configure, or check on the status of Wazuh processes. This script can enable or disable client-syslog, database logging, agentless configurations, integration with slack and pagerduty, and debug mode.
```

Comment:
Some versions ago the need for enabling client-syslog, database logging, agentless configurations, and integrations were removed. Change this sentence to:

```
The wazuh-control script is used to start, stop, configure, check on the status of Wazuh processes and enable the debug mode.
```